text/web-servers/0006-web-servers-rearchitecture.md

```diff
@@ -0,0 +1,270 @@
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/watchexec"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/node-engine"
+
+  [[order.group]]
+    id = "paketo-buildpacks/yarn"
+
+  [[order.group]]
+    id = "paketo-buildpacks/yarn-install"
+
+  [[order.group]]
+    id = "paketo-buildpacks/node-run-script"
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-dist"
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-zero-configuration"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-start"
+
+  [[order.group]]
+    id = "paketo-buildpacks/environment-variables"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/image-labels"
+    optional = true
+
+[[order]]
+
+  [[order.group]]
+    id = "paketo-buildpacks/ca-certificates"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/watchexec"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/node-engine"
+
+  [[order.group]]
+    id = "paketo-buildpacks/npm-install"
+
+  [[order.group]]
+    id = "paketo-buildpacks/node-run-script"
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-dist"
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-zero-configuration"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-start"
+
+  [[order.group]]
+    id = "paketo-buildpacks/environment-variables"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/image-labels"
+    optional = true
+
+[[order]]
+
+  [[order.group]]
+    id = "paketo-buildpacks/ca-certificates"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/watchexec"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/nginx-dist"
+
+  [[order.group]]
+    id = "paketo-buildpacks/nginx-zero-configuration"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/nginx-start"
+
+  [[order.group]]
+    id = "paketo-buildpacks/procfile"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/environment-variables"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/image-labels"
+    optional = true
+
+[[order]]
+
+  [[order.group]]
+    id = "paketo-buildpacks/ca-certificates"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/watchexec"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-dist"
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-zero-configuration"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/httpd-start"
+
+  [[order.group]]
+    id = "paketo-buildpacks/procfile"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/environment-variables"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-buildpacks/image-labels"
+    optional = true
+```
+
+## Rationale and Alternatives
+
+- Do nothing and ensure that the web server dependencies are ingestible by
```

Comment:
What would it look like to "web server dependencies are ingestible by third party buildpacks" without this refactor?